test-requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==22.3.0 ; python_version > '3.5'
+black==23.3.0 ; python_version > '3.9'
```

Comment:
I don't know if we really need the version checks here. The min we support is 3.9 anyway and it's going to be 3.10 after May.